phpcs.xml

```diff
@@ -16,6 +16,7 @@
 
     <exclude-pattern>modules/adfs/*</exclude-pattern>
     <exclude-pattern>public/assets/*</exclude-pattern>
+    <exclude-pattern>config/*</exclude-pattern>
```

Comment:
Why do we have to completely ignore the config-directory? Can't we suppress a specific issue if there is something we cannot fix?

Reply:
@tvdijen  We do not. I did this for consistency. Previously we were including the config directory but no file was taken into account since all files were template ones. Do you want me to include the config directory and retest?